release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -3982,3 +3982,30 @@ $ oc adm release info 4.13.37 --pullspecs
 [id="ocp-4-13-37-updating"]
 ==== Updating
 To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+[id="ocp-4-13-38"]
+=== RHSA-2024:1454 - {product-title} 4.13.38 bug fix and security update
+
+Issued: 2024-03-27
+
+{product-title} release 4.13.38, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:1454[RHSA-2024:1454] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:1456[RHSA-2024:1456] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.13.38 --pullspecs
+----
+
+[id="ocp-4-13-38-bug-fixes"]
+==== Bug fixes
+
+* Previously, machine sets that ran on Microsoft Azure regions with no availability zone support always created `AvailabilitySets` objects for Spot instances. This operation caused Spot instances to fail because the instances did not support availability sets. Now, machine sets do not create `AvailabilitySets` objects for Spot instances that operate in non-zonal configured regions. (link:https://issues.redhat.com/browse/OCPBUGS-29906[*OCPBUGS-29906*])
+
+* Previously, the `manila-csi-driver-controller-metrics` service had empty endpoints because an incorrect name for the app selector. With this release the app selector name is changed to `openstack-manila-csi` and the issue is fixed. (link:https://issues.redhat.com/browse/OCPBUGS-26224[*OCPBUGS-26224*])
```

Comment:
```suggestion
* Previously, the `manila-csi-driver-controller-metrics` service had empty endpoints because of an incorrect name for the app selector. With this release the app selector name is changed to `openstack-manila-csi` and the issue is fixed. (link:https://issues.redhat.com/browse/OCPBUGS-26224[*OCPBUGS-26224*])
```

Reply:
The above fix makes this unnecessary. Thanks!